### ISSUE TITLE
Couldn't get the native call queue: bridge configuration isn't available

### ISSUE THREAD

**Issue by vikeri** [facebook/react-native#7091]
_Mac, RN@master (887c275), Android_

Compiled the UIExampleProject with:

`./gradlew :Examples:UIExplorer:android:app:installDebug assembleRelease` in `react-native/`

Then sent this apk to my Android device and installed it. 

Thought I was going to get the app for offline use since there are no special instructions here for that [Running on Device - Android](http://facebook.github.io/react-native/releases/next/docs/running-on-device-android.html#content) as there are here [Running on Device - iOS](http://facebook.github.io/react-native/releases/next/docs/running-on-device-ios.html#content). 

Mostly I am here to pick up the trophy:

![screenshot_20160419-180732](https://cloud.githubusercontent.com/assets/846688/14667198/40205fa0-06de-11e6-8a2f-67012e9f5c47.png)

**Comment by dimitrovskif** [comment]
This is actually a very common error, but the error message has recently been changed:

https://github.com/facebook/react-native/commit/c24fae9544bb3f6a6313e9583c875591607bcefc

The previous error was a JS trace saying "__fbBatchedBridge" is not defined which is caused due to no packager running (or faulty packer running). You may find info here:

* https://github.com/facebook/react-native/issues/4952
* https://github.com/facebook/react-native/issues/6282
* https://github.com/facebook/react-native/issues/7022

**Comment by vikeri** [comment]
What worked was to `rm -rf node_modules && npm install`

**Comment by giantelk** [comment]
I deleted `/node_modules/` ran `npm install`, had to fix error where react-native 0.26.2 was looking for react 15.0.2, but it installed react 15.1., so I removed the `^` from this line in `package.json` ``"react": "^15.0.2"`. 

THEN... error still appears in emulator/app:
```
Couldn't get the native call queue: bridge configuration isn't available. this probably indicates there was an issue loading the JS bundle, e.g. it wasn't packaged into the app or was malformed. Check your logs ('adb logcat') for more information.
```
